pkg/tcpip/link/rawfile/blockingpoll_amd64_unsafe.go

```diff
@@ -22,11 +22,25 @@ package rawfile
 
 import (
 	"syscall"
-	_ "unsafe" // for go:linkname
+	"unsafe"
 )
 
+func BlockingPoll(fds *PollEvent, nfds int, timeout int64) (int, syscall.Errno) {
```

Comment:
FWIW, I think you could just change all of  the BlockingPoll callers to syscall.Timespec rather than adding a wrapper.

Reply:
Yes, for this specific usage case(tcpip/link/rawfile operation), it's reasonable to do it. 
Will fix it soon.

Thanks.

Reply:
fixed!